packages/dapp/src/components/header/NavMenu.tsx

```diff
@@ -40,20 +53,20 @@ const NavMenuLinksComponent: React.FunctionComponent<NavMenuCloseDrawerProp> = p
         <NavLink to="/parameterizer" {...mobileOnClickProp}>
           <NavLinkParameterizerText />
         </NavLink>
+        <NavLink href="https://civilfound.org/" target="_blank">
+          <NavLinkFoundationText />
+        </NavLink>
+        <NavLink href="https://learn.civil.co/constitution/" target="_blank">
+          <NavLinkConstitutionText />
+        </NavLink>
         <NavLink to="/contract-addresses" {...mobileOnClickProp}>
           <NavLinkContractAddressesText />
         </NavLink>
       </NavDropDown>
-      <NavLink to="/boosts" {...mobileOnClickProp}>
-        <NavLinkBoostsText />
-      </NavLink>
-      <NavLink href="https://civil.co/constitution/">
-        <NavLinkConstitutionText />
-      </NavLink>
-      <NavLink href="https://civilfound.org/">
-        <NavLinkFoundationText />
+      <NavLink href="https://learn.civil.co/boosts" target="_blank">
```

Comment:
Are there designs around the placement of this? Should it go under Boosts?  There's a potential space issue on smaller screens just before it collapses into mobile nav. If someone has a longer username it would likely get pushed off screen.